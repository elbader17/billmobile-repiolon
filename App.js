/**
 * Sample React Native App
 * https://github.com/facebook/react-native
 *
 * @format
 * @flow
 * @lint-ignore-every XPLATJSCOPYRIGHT1
 */

import React, {Component} from 'react';
import {Platform, StyleSheet, Text, View} from 'react-native';

import Amplify from 'aws-amplify';
import aws_exports from './src/aws-exports';
import Home from'./src/login/Home';

Amplify.configure(aws_exports);

type Props = {};
export default class App extends Component<Props> {
  render() {
    return (
      <View style={styles.container}>
        <Text> Welcome to React Native </Text>
        <Home/>
     
      </View>
    );
  }
}

const styles = StyleSheet.create({
  container: {
    flex: 5,
   
    backgroundColor: '#F5FCFF',
  },
  welcome: {
    fontSize: 20,
    textAlign: 'center',
    margin: 5,
  },
  instructions: {
    textAlign: 'center',
    color: '#333333',
    marginBottom: 5,
  },
});
