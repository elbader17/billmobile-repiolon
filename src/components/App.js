import * as React from 'react';
import { StatusBar} from 'react-native';
import { Provider } from 'react-redux';
import store from '../store';
import Amplify from 'aws-amplify';
import axios from 'axios';
import {
  API_HOST,
} from 'react-native-dotenv';
import aws_exports from '../constants/aws-exports';
import AppNavigator from './AppNavigator';

Amplify.configure(aws_exports);

axios.defaults.baseURL = API_HOST;

class App extends React.Component {
  componentDidMount() {
    StatusBar.setHidden(true);
  }
  render() {
    console.disableYellowBox = true;
    return (
      <Provider store={store}>
        <AppNavigator />
      </Provider>
    );
  }
}

export default App;

