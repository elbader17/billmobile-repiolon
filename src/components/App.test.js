import 'react-native';
import React from 'react';
import App from './App';
import renderer from 'react-test-renderer';

test('renders correctly App', () => {
  const component = renderer.create(<App />).toJSON();
  expect(component).toMatchSnapshot();
});

