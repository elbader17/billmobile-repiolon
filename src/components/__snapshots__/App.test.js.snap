// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly App 1`] = `
<View>
  <View>
    <View
      style={
        Object {
          "backgroundColor": "#fff",
          "borderColor": "#e3e3e3",
          "borderRadius": 3,
          "borderWidth": 1,
          "flexDirection": "row",
          "height": 40,
          "marginBottom": 5,
          "marginLeft": 10,
          "marginRight": 10,
          "marginTop": 5,
          "overflow": "hidden",
        }
      }
      testID="buttonGroup"
    >
      <View
        style={
          Object {
            "borderBottomLeftRadius": 3,
            "borderRightColor": "#bdc6cf",
            "borderRightWidth": 0,
            "borderTopLeftRadius": 3,
            "flex": 1,
          }
        }
      >
        <View
          accessible={true}
          isTVSelectable={true}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "flex": 1,
              "opacity": 1,
            }
          }
          testID="buttonGroupItem"
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "#2089dc",
                "flex": 1,
                "justifyContent": "center",
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#fff",
                  "fontSize": 16.25,
                  "fontWeight": "500",
                }
              }
              testID="buttonGroupItemText"
              theme={
                Object {
                  "colors": Object {
                    "disabled": "hsl(208, 8%, 90%)",
                    "divider": "#bcbbc1",
                    "error": "#ff190c",
                    "grey0": "#393e42",
                    "grey1": "#43484d",
                    "grey2": "#5e6977",
                    "grey3": "#86939e",
                    "grey4": "#bdc6cf",
                    "grey5": "#e1e8ee",
                    "greyOutline": "#bbb",
                    "platform": Object {
                      "android": Object {
                        "error": "#f44336",
                        "primary": "#2196f3",
                        "secondary": "#9C27B0",
                        "success": "#4caf50",
                        "warning": "#ffeb3b",
                      },
                      "ios": Object {
                        "error": "#ff3b30",
                        "primary": "#007aff",
                        "secondary": "#5856d6",
                        "success": "#4cd964",
                        "warning": "#ffcc00",
                      },
                    },
                    "primary": "#2089dc",
                    "searchBg": "#303337",
                    "secondary": "#8F0CE8",
                    "success": "#52c41a",
                    "warning": "#faad14",
                  },
                }
              }
            >
              Sign Up
            </Text>
          </View>
        </View>
      </View>
      <View
        style={
          Object {
            "borderBottomRightRadius": 3,
            "borderLeftColor": "#bdc6cf",
            "borderLeftWidth": 1,
            "borderTopRightRadius": 3,
            "flex": 1,
          }
        }
      >
        <View
          accessible={true}
          isTVSelectable={true}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "flex": 1,
              "opacity": 1,
            }
          }
          testID="buttonGroupItem"
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "flex": 1,
                "justifyContent": "center",
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#5e6977",
                  "fontSize": 16.25,
                  "fontWeight": "500",
                }
              }
              testID="buttonGroupItemText"
              theme={
                Object {
                  "colors": Object {
                    "disabled": "hsl(208, 8%, 90%)",
                    "divider": "#bcbbc1",
                    "error": "#ff190c",
                    "grey0": "#393e42",
                    "grey1": "#43484d",
                    "grey2": "#5e6977",
                    "grey3": "#86939e",
                    "grey4": "#bdc6cf",
                    "grey5": "#e1e8ee",
                    "greyOutline": "#bbb",
                    "platform": Object {
                      "android": Object {
                        "error": "#f44336",
                        "primary": "#2196f3",
                        "secondary": "#9C27B0",
                        "success": "#4caf50",
                        "warning": "#ffeb3b",
                      },
                      "ios": Object {
                        "error": "#ff3b30",
                        "primary": "#007aff",
                        "secondary": "#5856d6",
                        "success": "#4cd964",
                        "warning": "#ffcc00",
                      },
                    },
                    "primary": "#2089dc",
                    "searchBg": "#303337",
                    "secondary": "#8F0CE8",
                    "success": "#52c41a",
                    "warning": "#faad14",
                  },
                }
              }
            >
              Sign In
            </Text>
          </View>
        </View>
      </View>
    </View>
    <View>
      <View
        style={
          Object {
            "margin": 20,
          }
        }
      >
        <View
          style={
            Object {
              "paddingHorizontal": 10,
              "width": "100%",
            }
          }
        >
          <Text
            style={
              Object {
                "color": "#86939e",
                "fontSize": 16,
                "fontWeight": "bold",
              }
            }
          >
            Nombre
          </Text>
          <View
            style={
              Object {
                "alignItems": "center",
                "borderBottomWidth": 1,
                "borderColor": "#86939e",
                "flexDirection": "row",
                "transform": Array [
                  Object {
                    "translateX": 0,
                  },
                ],
              }
            }
          >
            <TextInput
              allowFontScaling={true}
              onChangeText={[Function]}
              placeholder="Nombre"
              style={
                Object {
                  "alignSelf": "center",
                  "color": "black",
                  "flex": 1,
                  "fontSize": 18,
                  "minHeight": 40,
                }
              }
              testID="RNE__Input__text-input"
              underlineColorAndroid="transparent"
            />
          </View>
        </View>
        <View
          style={
            Object {
              "paddingHorizontal": 10,
              "width": "100%",
            }
          }
        >
          <Text
            style={
              Object {
                "color": "#86939e",
                "fontSize": 16,
                "fontWeight": "bold",
              }
            }
          >
            Email
          </Text>
          <View
            style={
              Object {
                "alignItems": "center",
                "borderBottomWidth": 1,
                "borderColor": "#86939e",
                "flexDirection": "row",
                "transform": Array [
                  Object {
                    "translateX": 0,
                  },
                ],
              }
            }
          >
            <TextInput
              allowFontScaling={true}
              editable={true}
              onChangeText={[Function]}
              onRef={[Function]}
              placeholder="usuario@email.com"
              returnKeyType="next"
              style={
                Object {
                  "alignSelf": "center",
                  "color": "black",
                  "flex": 1,
                  "fontSize": 18,
                  "minHeight": 40,
                }
              }
              testID="RNE__Input__text-input"
              underlineColorAndroid="transparent"
              value=""
            />
          </View>
        </View>
        <View>
          <View
            onResponderRelease={[Function]}
            onStartShouldSetResponder={[Function]}
            pointerEvents="auto"
          >
            <View
              style={
                Object {
                  "backgroundColor": "transparent",
                  "borderBottomColor": "rgba(0, 0, 0, 0.38)",
                  "borderBottomWidth": 0.5,
                  "height": 64,
                  "paddingBottom": 8,
                  "paddingTop": 32,
                }
              }
            >
              <View
                style={
                  Object {
                    "position": "absolute",
                    "top": 16,
                  }
                }
              >
                <Text
                  active={true}
                  animationDuration={225}
                  errored={false}
                  focused={false}
                  numberOfLines={1}
                  style={
                    Object {
                      "color": "rgba(0, 0, 0, 0.38)",
                      "fontSize": 12,
                    }
                  }
                >
                  Password
                </Text>
              </View>
              <View
                style={
                  Object {
                    "flexDirection": "row",
                  }
                }
              >
                <TextInput
                  allowFontScaling={true}
                  autoCapitalize="sentences"
                  blurOnSubmit={true}
                  disableFullscreenUI={true}
                  editable={true}
                  iconSize={25}
                  leftIcon={
                    Object {
                      "name": "lock",
                      "type": "font-awesome",
                    }
                  }
                  onBlur={[Function]}
                  onChange={[Function]}
                  onChangeText={[Function]}
                  onContentSizeChange={[Function]}
                  onFocus={[Function]}
                  placeholder="Aa@-1234"
                  secureTextEntry={true}
                  selectionColor="rgb(0, 145, 234)"
                  style={
                    Array [
                      Object {
                        "flex": 1,
                        "margin": 0,
                        "padding": 0,
                        "top": 2,
                      },
                      Object {
                        "color": "rgba(0, 0, 0, .87)",
                        "fontSize": 16,
                        "height": 24,
                      },
                      undefined,
                    ]
                  }
                  underlineColorAndroid="transparent"
                  value=""
                />
              </View>
            </View>
            <View
              style={
                Object {
                  "flexDirection": "row",
                  "height": 8,
                }
              }
            >
              <View
                style={
                  Object {
                    "flex": 1,
                  }
                }
              >
                <View
                  style={
                    Object {
                      "bottom": 0,
                      "left": 0,
                      "paddingVertical": 4,
                      "position": "absolute",
                      "right": 0,
                      "top": 0,
                    }
                  }
                >
                  <Text
                    numberOfLines={1}
                    style={
                      Object {
                        "backgroundColor": "transparent",
                        "color": "rgb(213, 0, 0)",
                        "fontSize": 0,
                        "opacity": 0,
                      }
                    }
                  />
                </View>
                <View
                  style={
                    Object {
                      "bottom": 0,
                      "left": 0,
                      "paddingVertical": 4,
                      "position": "absolute",
                      "right": 0,
                      "top": 0,
                    }
                  }
                >
                  <Text
                    numberOfLines={1}
                    style={
                      Object {
                        "backgroundColor": "transparent",
                        "color": "rgba(0, 0, 0, .38)",
                        "fontSize": 12,
                        "opacity": 1,
                      }
                    }
                  />
                </View>
              </View>
            </View>
          </View>
          <Text
            allowFontScaling={false}
            onPress={[Function]}
            style={
              Array [
                Object {
                  "color": undefined,
                  "fontSize": 25,
                },
                Object {
                  "position": "absolute",
                  "right": 0,
                  "top": 33,
                },
                Object {
                  "fontFamily": "Material Icons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
              ]
            }
          >
            
          </Text>
        </View>
        <View>
          <View
            onResponderRelease={[Function]}
            onStartShouldSetResponder={[Function]}
            pointerEvents="auto"
          >
            <View
              style={
                Object {
                  "backgroundColor": "transparent",
                  "borderBottomColor": "rgba(0, 0, 0, 0.38)",
                  "borderBottomWidth": 0.5,
                  "height": 64,
                  "paddingBottom": 8,
                  "paddingTop": 32,
                }
              }
            >
              <View
                style={
                  Object {
                    "position": "absolute",
                    "top": 16,
                  }
                }
              >
                <Text
                  active={true}
                  animationDuration={225}
                  errored={false}
                  focused={false}
                  numberOfLines={1}
                  style={
                    Object {
                      "color": "rgba(0, 0, 0, 0.38)",
                      "fontSize": 12,
                    }
                  }
                >
                  Password
                </Text>
              </View>
              <View
                style={
                  Object {
                    "flexDirection": "row",
                  }
                }
              >
                <TextInput
                  allowFontScaling={true}
                  autoCapitalize="sentences"
                  blurOnSubmit={true}
                  disableFullscreenUI={true}
                  editable={true}
                  iconSize={25}
                  leftIcon={
                    Object {
                      "name": "lock",
                      "type": "font-awesome",
                    }
                  }
                  onBlur={[Function]}
                  onChange={[Function]}
                  onChangeText={[Function]}
                  onContentSizeChange={[Function]}
                  onFocus={[Function]}
                  placeholder="Aa@-1234"
                  secureTextEntry={true}
                  selectionColor="rgb(0, 145, 234)"
                  style={
                    Array [
                      Object {
                        "flex": 1,
                        "margin": 0,
                        "padding": 0,
                        "top": 2,
                      },
                      Object {
                        "color": "rgba(0, 0, 0, .87)",
                        "fontSize": 16,
                        "height": 24,
                      },
                      undefined,
                    ]
                  }
                  underlineColorAndroid="transparent"
                  value=""
                />
              </View>
            </View>
            <View
              style={
                Object {
                  "flexDirection": "row",
                  "height": 8,
                }
              }
            >
              <View
                style={
                  Object {
                    "flex": 1,
                  }
                }
              >
                <View
                  style={
                    Object {
                      "bottom": 0,
                      "left": 0,
                      "paddingVertical": 4,
                      "position": "absolute",
                      "right": 0,
                      "top": 0,
                    }
                  }
                >
                  <Text
                    numberOfLines={1}
                    style={
                      Object {
                        "backgroundColor": "transparent",
                        "color": "rgb(213, 0, 0)",
                        "fontSize": 0,
                        "opacity": 0,
                      }
                    }
                  />
                </View>
                <View
                  style={
                    Object {
                      "bottom": 0,
                      "left": 0,
                      "paddingVertical": 4,
                      "position": "absolute",
                      "right": 0,
                      "top": 0,
                    }
                  }
                >
                  <Text
                    numberOfLines={1}
                    style={
                      Object {
                        "backgroundColor": "transparent",
                        "color": "rgba(0, 0, 0, .38)",
                        "fontSize": 12,
                        "opacity": 1,
                      }
                    }
                  />
                </View>
              </View>
            </View>
          </View>
          <Text
            allowFontScaling={false}
            onPress={[Function]}
            style={
              Array [
                Object {
                  "color": undefined,
                  "fontSize": 25,
                },
                Object {
                  "position": "absolute",
                  "right": 0,
                  "top": 33,
                },
                Object {
                  "fontFamily": "Material Icons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
              ]
            }
          >
            
          </Text>
        </View>
        <View
          style={
            Object {
              "borderRadius": 3,
            }
          }
        >
          <View
            accessible={true}
            isTVSelectable={true}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "opacity": 1,
              }
            }
            testID="submitSignUp"
          >
            <View
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "hsl(208, 8%, 90%)",
                  "borderColor": "#2089dc",
                  "borderRadius": 3,
                  "borderWidth": 0,
                  "flexDirection": "row",
                  "justifyContent": "center",
                  "padding": 8,
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": Object {
                      "color": Array [
                        208,
                        8,
                        63,
                      ],
                      "model": "hsl",
                      "valpha": 1,
                    },
                    "fontSize": 18,
                    "paddingBottom": 1,
                    "paddingTop": 2,
                    "textAlign": "center",
                  }
                }
              >
                Submit
              </Text>
            </View>
          </View>
        </View>
        <Modal
          hardwareAccelerated={false}
          visible={false}
        >
          <View>
            <View>
              <View
                style={
                  Object {
                    "paddingHorizontal": 10,
                    "width": "100%",
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "color": "#86939e",
                      "fontSize": 16,
                      "fontWeight": "bold",
                    }
                  }
                >
                  Email
                </Text>
                <View
                  style={
                    Object {
                      "alignItems": "center",
                      "borderBottomWidth": 1,
                      "borderColor": "#86939e",
                      "flexDirection": "row",
                      "transform": Array [
                        Object {
                          "translateX": 0,
                        },
                      ],
                    }
                  }
                >
                  <View
                    style={
                      Object {
                        "alignItems": "center",
                        "height": 40,
                        "justifyContent": "center",
                        "marginLeft": 15,
                      }
                    }
                  >
                    <View>
                      <View
                        style={
                          Object {
                            "alignItems": "center",
                            "backgroundColor": "transparent",
                            "justifyContent": "center",
                          }
                        }
                        theme={
                          Object {
                            "colors": Object {
                              "disabled": "hsl(208, 8%, 90%)",
                              "divider": "#bcbbc1",
                              "error": "#ff190c",
                              "grey0": "#393e42",
                              "grey1": "#43484d",
                              "grey2": "#5e6977",
                              "grey3": "#86939e",
                              "grey4": "#bdc6cf",
                              "grey5": "#e1e8ee",
                              "greyOutline": "#bbb",
                              "platform": Object {
                                "android": Object {
                                  "error": "#f44336",
                                  "primary": "#2196f3",
                                  "secondary": "#9C27B0",
                                  "success": "#4caf50",
                                  "warning": "#ffeb3b",
                                },
                                "ios": Object {
                                  "error": "#ff3b30",
                                  "primary": "#007aff",
                                  "secondary": "#5856d6",
                                  "success": "#4cd964",
                                  "warning": "#ffcc00",
                                },
                              },
                              "primary": "#2089dc",
                              "searchBg": "#303337",
                              "secondary": "#8F0CE8",
                              "success": "#52c41a",
                              "warning": "#faad14",
                            },
                          }
                        }
                        underlayColor="white"
                      >
                        <Text
                          allowFontScaling={false}
                          style={
                            Array [
                              Object {
                                "color": "black",
                                "fontSize": 24,
                              },
                              Object {
                                "backgroundColor": "transparent",
                              },
                              Object {
                                "fontFamily": "FontAwesome",
                                "fontStyle": "normal",
                                "fontWeight": "normal",
                              },
                              Object {},
                            ]
                          }
                          testID="iconIcon"
                        >
                          
                        </Text>
                      </View>
                    </View>
                  </View>
                  <TextInput
                    allowFontScaling={true}
                    onChangeText={[Function]}
                    style={
                      Object {
                        "alignSelf": "center",
                        "color": "black",
                        "flex": 1,
                        "fontSize": 18,
                        "minHeight": 40,
                      }
                    }
                    testID="RNE__Input__text-input"
                    underlineColorAndroid="transparent"
                  />
                </View>
              </View>
              <View
                style={
                  Object {
                    "paddingHorizontal": 10,
                    "width": "100%",
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "color": "#86939e",
                      "fontSize": 16,
                      "fontWeight": "bold",
                    }
                  }
                >
                  Codigo de confirmación
                </Text>
                <View
                  style={
                    Object {
                      "alignItems": "center",
                      "borderBottomWidth": 1,
                      "borderColor": "#86939e",
                      "flexDirection": "row",
                      "transform": Array [
                        Object {
                          "translateX": 0,
                        },
                      ],
                    }
                  }
                >
                  <View
                    style={
                      Object {
                        "alignItems": "center",
                        "height": 40,
                        "justifyContent": "center",
                        "marginLeft": 15,
                      }
                    }
                  >
                    <View>
                      <View
                        style={
                          Object {
                            "alignItems": "center",
                            "backgroundColor": "transparent",
                            "justifyContent": "center",
                          }
                        }
                        theme={
                          Object {
                            "colors": Object {
                              "disabled": "hsl(208, 8%, 90%)",
                              "divider": "#bcbbc1",
                              "error": "#ff190c",
                              "grey0": "#393e42",
                              "grey1": "#43484d",
                              "grey2": "#5e6977",
                              "grey3": "#86939e",
                              "grey4": "#bdc6cf",
                              "grey5": "#e1e8ee",
                              "greyOutline": "#bbb",
                              "platform": Object {
                                "android": Object {
                                  "error": "#f44336",
                                  "primary": "#2196f3",
                                  "secondary": "#9C27B0",
                                  "success": "#4caf50",
                                  "warning": "#ffeb3b",
                                },
                                "ios": Object {
                                  "error": "#ff3b30",
                                  "primary": "#007aff",
                                  "secondary": "#5856d6",
                                  "success": "#4cd964",
                                  "warning": "#ffcc00",
                                },
                              },
                              "primary": "#2089dc",
                              "searchBg": "#303337",
                              "secondary": "#8F0CE8",
                              "success": "#52c41a",
                              "warning": "#faad14",
                            },
                          }
                        }
                        underlayColor="white"
                      >
                        <Text
                          allowFontScaling={false}
                          style={
                            Array [
                              Object {
                                "color": "black",
                                "fontSize": 24,
                              },
                              Object {
                                "backgroundColor": "transparent",
                              },
                              Object {
                                "fontFamily": "FontAwesome",
                                "fontStyle": "normal",
                                "fontWeight": "normal",
                              },
                              Object {},
                            ]
                          }
                          testID="iconIcon"
                        >
                          
                        </Text>
                      </View>
                    </View>
                  </View>
                  <TextInput
                    allowFontScaling={true}
                    onChangeText={[Function]}
                    style={
                      Object {
                        "alignSelf": "center",
                        "color": "black",
                        "flex": 1,
                        "fontSize": 18,
                        "minHeight": 40,
                      }
                    }
                    testID="RNE__Input__text-input"
                    underlineColorAndroid="transparent"
                  />
                </View>
              </View>
              <View
                style={
                  Object {
                    "borderRadius": 3,
                  }
                }
              >
                <View
                  accessible={true}
                  isTVSelectable={true}
                  onResponderGrant={[Function]}
                  onResponderMove={[Function]}
                  onResponderRelease={[Function]}
                  onResponderTerminate={[Function]}
                  onResponderTerminationRequest={[Function]}
                  onStartShouldSetResponder={[Function]}
                  style={
                    Object {
                      "opacity": 1,
                    }
                  }
                  testID="submitConfirmation"
                >
                  <View
                    style={
                      Object {
                        "alignItems": "center",
                        "backgroundColor": "#2089dc",
                        "borderColor": "#2089dc",
                        "borderRadius": 3,
                        "borderWidth": 0,
                        "flexDirection": "row",
                        "justifyContent": "center",
                        "padding": 8,
                      }
                    }
                  >
                    <Text
                      style={
                        Object {
                          "color": "white",
                          "fontSize": 18,
                          "paddingBottom": 1,
                          "paddingTop": 2,
                          "textAlign": "center",
                        }
                      }
                    >
                      Submit
                    </Text>
                  </View>
                </View>
              </View>
            </View>
          </View>
        </Modal>
      </View>
    </View>
  </View>
</View>
`;
